src/routes/__layout.svelte

```diff
@@ -23,7 +23,7 @@
 
     loadUser();
     loadCluster();
-    loadSettings();
+    await loadSettings();
```

Comment:
We probably only want to force this to be syncronous when we actually need the data so there should probably be a better mechanism to do that.

Reply:
removed await and made the navigation reactively wait until there is a value